src/XrdCl/XrdClUtils.hh

```diff
@@ -56,25 +57,7 @@ namespace XrdCl
                                const std::string &input,
                                const std::string &delimiter )
       {
-        size_t start  = 0;
-        size_t end    = 0;
-        size_t length = 0;
-
-        do
-        {
-          end = input.find( delimiter, start );
-
-          if( end != std::string::npos )
-            length = end - start;
-          else
-            length = input.length() - start;
-
-          if( length )
-            result.push_back( input.substr( start, length ) );
-
-          start = end + delimiter.size();
-        }
-        while( end != std::string::npos );
+        XrdOucUtils::splitString(result,input,delimiter);
```

Comment:
Likely add a comment providing some history on this with a mention that at some point it would be good to collapse this change (i.e., eliminate the middle man call).